src/aics-image-viewer/components/App/index.tsx

```diff
@@ -1159,6 +1160,22 @@ export default class App extends React.Component<AppProps, AppState> {
     return { x: 0, y: 0, z: 0 };
   }
 
+  getMetadata(): MetadataRecord {
+    const { metadata, metadataFormatter } = this.props;
+    const { image } = this.state;
+
+    let imageMetadata = image?.imageMetadata as MetadataRecord;
```

Comment:
Should we put the MetadataRecord type down in volume-viewer?

Reply:
Potentially, though it matters to what extent `MetadataRecord` is a contract about what values are allowable in metadata objects vs. just a convenience to easily keep values well-typed while `MetadataViewer` is recursively rendering them. I can't think of many values that would realistically appear in a metadata object but wouldn't be covered by `MetadataRecord`, even if TypeScript can come up with edge cases. And even if it were in volume-viewer, it would still be reading external data and making guesses about its type. I don't have much of a problem making that guess here instead.

Hm - I was leaning yes when I started, then talked myself into no as I wrote.